VMEncryption/main/handle.py

```diff
@@ -509,19 +509,35 @@ def enable():
         hutil.do_parse_context('Enable')
         logger.log('Enabling extension')
 
+        public_settings = get_public_settings()
+        logger.log('Public settings:\n{0}'.format(json.dumps(public_settings, sort_keys=True, indent=4)))
+        cutil = CheckUtil(logger)
+
+        # run fatal prechecks, report error if exceptions are caught
+        try:
+            cutil.precheck_for_fatal_failures(public_settings)
+        except Exception as e:
+            logger.log("PRECHECK: Fatal Exception thrown during precheck")
+            logger.log(traceback.format_exc())
+            msg = e.message
+            hutil.do_exit(exit_code=0,
+                          operation='Enable',
+                          status=CommonVariables.extension_error_status,
+                          code=(CommonVariables.configuration_error),
+                          message=msg)
+
+        hutil.disk_util.log_lsblk_output()
```

Comment:
log this after finishing encryption as well. That way we know what got encrypted and mounted. Update encryption settings should also print this.